### ISSUE TITLE
ConfigurationUtil warning for duplicate registrations

### ISSUE THREAD

**Issue by alfalfa6945** [LoogleCZ/FS17-FillableConfiguration#1]
If two mods are running the FillableConfiguration script, there will be a log warning when the second mod loads as it's trying to register an existing configuration (fillConf). Would it be possible to add in a sanity check?

```lua
if not ConfigurationUtil.configurations.fillConf then
	ConfigurationUtil.registerConfigurationType("fillConf", g_i18n:getText("l10n_fillableConfiguration"), nil, nil, nil, ConfigurationUtil.SELECTOR_MULTIOPTION);
end;
```

**Comment by LoogleCZ** [comment]
Hi! Thank you for reporting this issue! I did not notice this before.

On Sunday/Monday I will add this check to the script.

Again thanks for the report
Cheers
Martin Fabík

**Comment by alfalfa6945** [comment]
It was a minor oversight. Thank YOU for taking the time to make this script to begin with

**Comment by LoogleCZ** [comment]
This issue is closed in release 1.0.1

Available here: https://github.com/LoogleCZ/FS17-FillableConfiguration/releases/tag/v1.0.1

Thanks @alfalfa6945 for reporting this issue.
